common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -977,6 +977,10 @@ def student_view_data(self, context=None):
         encoded_videos = {}
         val_video_data = {}
 
+        all_sources = self.html5_sources or []
+        if self.source:
+            all_sources.insert(0, self.source)
```

Comment:
Maybe note with a comment that `source` is a deprecated field. Also, the former behavior here (line 1010 before this change) was to only fall back to `source` if `html5_sources` was empty, so you should probably preserve that behavior by appending `source` to the  end, if it's set at all, rather than the beginning.

Reply:
Oh aye -- done with https://github.com/edx/edx-platform/pull/16590/files#diff-2386122d0b4952fd587ad8fa9556ef8dR981.